docker/develop/Dockerfile

```diff
@@ -194,11 +194,18 @@ RUN git clone git://github.com/warchant/ed25519.git /tmp/ed25519; \
     ldconfig; \
     rm -rf /tmp/ed25519
 
-RUN adduser --disabled-password  iroha-ci --uid 6535
+# non-interactive adduser
+#   -m = create home dir
+#   -s = set default shell
+#   iroha-ci = username
+#   -u = userid, default for Ubuntu is 1000
+#   -U = create a group same as username
+#   no password
+RUN useradd -ms /bin/bash iroha-ci -u 1000 -U
```

Comment:
Prefer `useradd --no-create-home --shell /bin/bash iroha-ci --uid 1000 --user-group` instead and remove comments plz